src/profiler.h

```diff
@@ -64,6 +65,7 @@ class Profiler {
     ThreadFilter _thread_filter;
     CallTraceStorage _call_trace_storage;
     FlightRecorder _jfr;
+    ProtoBuffer _otlp_buffer{0};
```

Comment:
It's not unsed outside `dumpOtlp`, can we make it local?

Reply:
I think it's acceptable. Let's make a buffer local with a reasonable initial size.

Reply:
cbb5604f3ccd1e6df8e44f98ecf526ef2ee271c4